stdlib/datetime.pyi

```diff
@@ -55,7 +55,7 @@ class date:
     @property
     def day(self) -> int: ...
     def ctime(self) -> str: ...
-    def strftime(self, fmt: str) -> str: ...
+    def strftime(self, format: str) -> str: ...
```

Comment:
```suggestion
    def strftime(self, __format: str) -> str: ...
```
Let's make it positional-only as discussed.

Reply:
Quoting https://bugs.python.org/msg373421:
> I think in typeshed they can safely change from `fmt` to `format` even today (which would almost certainly be more accurate to end user use cases).

What do you have against this?

Reply:
Type checking is all about warning users about situations where their code could lead to unexpected errors. Passing a keyword argument to this parameter will currently cause code to raise an error sometimes (but not all the time, or even most of the time), as not all systems can be guaranteed to use the C implementation of `datetime` (this is partly why the Python implementation exists alongside the C implementation). If a user passes a positional argument to this parameter, however, then there is no possibility of an error. So, it seems correct that, as the situation currently stands, a type-checker should raise an error if a keyword argument is passed to this parameter, as doing so could in some situations lead to unexpected errors.

Reply:
It's definitely *better than the current situation* to have the parameter named `format`, but I agree that the best option for now is to have it as a positional-only parameter in the stub.